h5g_group_test.go

```diff
@@ -28,6 +32,14 @@ func TestGroup(t *testing.T) {
 		t.Errorf("wrong Name for group: want %q, got %q", "/foo", g1.Name())
 	}
 
+	if !f.LinkExists("/foo") {
+		t.Fatalf("err: %s", "/foo should exist at this time")
```

Comment:
Please remove the "err:" prefixes on these test failures. They don't add anything and they are misleading.

```
t.Error(`unexpected "/foo" group link`)
```

Reply:
Shouldn't this line 36 be 

> t.Error(`expected "/foo" group link`)

Reply:
Yes. Thanks.